src/native/corehost/fxr/fx_resolver.cpp

```diff
@@ -252,6 +259,12 @@ namespace
                     fx_ver_t ver;
                     if (fx_ver_t::parse(version, &ver, false))
                     {
+                        if (std::find(disabled_versions.begin(), disabled_versions.end(), version) != disabled_versions.end())
+                        {
+                            trace::verbose(_X("Ignoring disabled version [%s]"), version.c_str());
```

Comment:
Do we want to record this in the block of text we share emit for the current "status"? It is the output the SDK parses and users can see what is installed and what will occur.

Reply:
Unclear which I mean. Just thinking about general user confirmation/feedback when it is set.